keadm/cmd/keadm/app/cmd/debug/check.go

```diff
@@ -11,13 +11,13 @@ import (
 	"strings"
 	"time"
 
+	"github.com/kubeedge/kubeedge/common/constants"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/util"
```

Comment:
please format the import order

Reply:
hi, now i am use the "goimports" to save the file to keep the import order.
What is the format style is good for the ci?

Reply:
@gy95 
thanks for explain .
so kubeage is used 'gofmt' to format the code , note 'goimport'.
Fix it